contracts/Commitment/ITokenOffering.sol

```diff
@@ -9,7 +9,9 @@ contract ITokenOffering {
     ////////////////////////
 
     /// on every investment transaction
-    /// `investor` invested `amount` in `paymentToken` currency which was converted to `eurEquivalent` that purchases `purchasedAmount` of `ofToken`
+    /// `investor` invested `amount` in `paymentToken` currency which was
+    /// converted to `eurEquivalent` that purchases `purchasedAmount` of
+    /// `ofToken`.
     event LogFundsInvested(
```

Comment:
Why do we keep this interface if it is not used? To compile `LockedAccount`? It requires this interface to work.

Reply:
Yes, it's there because it's used in LockedAccount.